CollisionDetectionSystem/Interfaces/ITransponderReceiver.cs

```diff
@@ -1,13 +1,14 @@
 ﻿using System;
+using System.Collections.Generic;
 
 namespace CollisionDetectionSystem
 {
-	public delegate void DataDel(TransponderData data);
+	public delegate void DataDel( List<TransponderData> data);
 
 	public interface ITransponderReceiver
 	{
-		void ReceiveData(TransponderData data);
-		void PrepareDataForPost(TransponderData data);
+		void ReceiveData(List<TransponderData> data);
+		void PrepareDataForPost( List<TransponderData> data);
 		event DataDel PostDataEvent;
```

Comment:
Hey Fred, I don't understand why we want a List here.  This TransponderReceiver is suppose to receive just one transponderData at a time.  This in the real world would be the transponder getting one ping from a plane.  It would not get a list.  I will checkout out your branch and try to understand a bit more.